docs/devguide/metricset-details.asciidoc

```diff
@@ -104,7 +112,7 @@ Here an example for the `fields.yml` file from the MySQL module.
 include::../../metricbeat/module/mysql/_meta/fields.yml[]
 ----
 
-There is another `fields.yml` file under `module/{module}/{metricset}/_meta/fields.yml` that contains all fields retrieved
+The last `fields.yml` file under `module/{module}/{metricset}/_meta/fields.yml` that contains all fields retrieved
```

Comment:
The sentence structure here is not parallel with the previous bullet item, but that problem will be fixed if you change the structure as I described previously. 

If you decide to punt, change this bullet item to use a parallel sentence structure: "The xyz is under....and it contains..."

Reply:
Fixed! :)